### ISSUE TITLE
CHORE: Remove unnecessary "foss" prefixes

### ISSUE THREAD

**Issue by rahulporuri** [fossunited/fossunited#487]
### Issue description

There are a number of python modules which unnecessarily have `foss` prefixes e.g. in their names, in the Python classes, in the doctype IDs, etc. The prefix doesn't provide any meaningful value because the python package is already `fossunited` so everything inside is implicitly or explicitly prefixed with `fossunited`.

For example, the below code becomes

```
from fossunited.foss_profiles.doctype.foss_user_profile_education.foss_user_profile_education import (          
    FOSSUserProfileEducation,
)
from fossunited.foss_profiles.doctype.foss_user_profile_work_experience.foss_user_profile_work_experience import (
    FOSSUserProfileWorkExperience,
)
from fossunited.foss_profiles.doctype.foss_user_projects.foss_user_projects import (
    FOSSUserProjects,
)
from fossunited.foss_profiles.doctype.foss_user_skill_multiselect.foss_user_skill_multiselect import (
    FOSSUserSkillMultiselect,
)
```

becomes 

```
from fossunited.user_profiles.doctype.user_profile_education.user_profile_education import (          
    UserProfileEducation,
)
from fossunited.user_profiles.doctype.user_profile_work_experience.user_profile_work_experience import (
    UserProfileWorkExperience,
)
from fossunited.user_profiles.doctype.user_projects.user_projects import (
    UserProjects,
)
from fossunited.user_profiles.doctype.user_skill_multiselect.user_skill_multiselect import (
    UserSkillMultiselect,
)
```

Similarly,

```
from fossunited.chapters.doctype.foss_event_community_partner.foss_event_community_partner import (
    FOSSEventCommunityPartner,
)
from fossunited.fossunited.doctype.foss_event_schedule.foss_event_schedule import (
    FOSSEventSchedule,
)
from fossunited.fossunited.doctype.foss_event_sponsor.foss_event_sponsor import (
    FOSSEventSponsor,
)
```

becomes

```
from fossunited.chapters.doctype.event_community_partner.event_community_partner import (
    EventCommunityPartner,
)
from fossunited.fossunited.doctype.event_schedule.event_schedule import (
    EventSchedule,
)
from fossunited.fossunited.doctype.event_sponsor.event_sponsor import (
    EventSponsor,
)
```

**Comment by RiyaMathew-11** [comment]
We would basically have to rename the doctypes, and field-names too if they come across it.

**Comment by harshtandiya** [comment]
Not touching this without some help from Frappe folks

**Comment by rahulporuri** [comment]
> We would basically have to rename the doctypes, and field-names too if they come across it.

That feels non-trivial but yes, that would be ideal.

> Not touching this without some help from Frappe folks

@NagariaHussain thoughts?

This issue only pertains to
* rename python modules and subpackages i.e. `fossunited.foss_profiles` becomes `fossunited.profiles` and `foss_event_community_partner.py` module becomes `event_community_partner.py`
* if possible, rename the Python classes (DocTypes) i.e. `FOSSHackathon` just becomes `Hackathon`

I don't know the frappe framework enough but given my understanding of Python in general, these should not break anything fundamental.

**Comment by mngshm** [comment]
We can write a shell script to do the mass change. But yes, I think this big of a change in the codebase file names should be done under the guidance of Frappe folks just to be safe.

**Comment by rahulporuri** [comment]
> We can write a shell script to do the mass change.

Remember that this isn't just a file rename, we also need to find and update the imports. I'm sure it's possible with scripting but most IDEs enable renaming files while automatically updating import statements.

**Comment by mngshm** [comment]
> Remember that this isn't just a file rename, we also need to find and update the imports

Yup, I'm aware of this. My recent PR did the same (but not with the file renames). The [PR ](https://github.com/fossunited/fossunited/pull/493), in this changes were made only through the command line, I haven't opened editor even once. 

>most IDEs enable renaming files while automatically updating import statements.

hmm, I think we need to discuss on finding the best approach.

**Comment by idlip** [comment]
Ok needs big refactor.

Simple way I can think of is:

1. Grep replace in dashboard code
2. Write migration code for name changes.

This can be experimented step by step for each doctype

But still unsure, gotta have a word with frappe!

**Comment by nimiverma** [comment]
Hi, I am new to open source and I would like to work on this issue. 
I have already set up the project locally and will submit a PR soon.
Please let me know if there are any guidelines I should follow.
